### PR TITLE
[Behat] Added initial search test

### PR DESCRIPTION
**Pull request by bogusez**
https://jira.ez.no/browse/EZP-31609

I have added initial search test

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: behat_suites.yml [hunk 1]
```diff
@@ -35,6 +35,8 @@ adminui:
                 - EzSystems\EzPlatformAdminUi\Behat\BusinessContext\DashboardContext
                 - EzSystems\BehatBundle\Context\Object\ContentTypeContext
                 - EzSystems\BehatBundle\Context\Api\TestContext
+                - EzSystems\EzPlatformAdminUi\Behat\BusinessContext\SearchContext
```
Comment (mnocon) [inline behat_suites.yml:38]: I'd continue grouping the Context classes listed here by the bundle they come from

File: features/standard/SearchContent.feature [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Feature: Searching for a Content
+  As an administrator
+  I want to search for content items.
```
Comment (mnocon) [inline features/standard/SearchContent.feature:3]:
```suggestion
  I want to search for Content Items.
```

File: features/standard/SearchContent.feature [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+  @javascript @admin @common
+  Scenario: Content can be searched
+
```
[the remark this answers is not in the document]
Comment (bogusez) [reply features/standard/SearchContent.feature]: Done, thx

File: features/standard/SearchContent.feature (hunk 2 above)
Comment (bogusez) [reply features/standard/SearchContent.feature]: Done, thx

File: behat_suites.yml (hunk 1 above)
Comment (bogusez) [reply behat_suites.yml:36]: Done, thx

File: features/standard/SearchContent.feature [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Feature: Searching for a Content
```
Comment (micszo) [inline features/standard/SearchContent.feature:1]: Nitpick: I would changes this to "Searching for Content" or "Searching for a Content item"

File: features/standard/SearchContent.feature [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  As an administrator
+  I want to search for Content items.
+
+  @javascript @admin @common
+  Scenario: Content can be searched
```
Comment (micszo) [inline features/standard/SearchContent.feature:6]: Nitpick: "Content can be searched for"

File: features/standard/SearchContent.feature [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    Given I create "Folder" Content items in root in "eng-GB"
+      | name              | short_name          |
+      | Searched folder   | Searched folder     |
+    And I am logged as "admin"
+    And I go to "Content structure" in "Content" tab
+    And I click on the left menu bar button "Search"
+    When I search for a Content named "Searched folder"
+    Then I should see in search results an item named "Searched folder"
```
Comment (micszo) [inline features/standard/SearchContent.feature:14]: Is a newline missing here?

File: features/standard/SearchContent.feature (hunk 4 above)
Comment (bogusez) [reply features/standard/SearchContent.feature]: Done, thx

File: features/standard/SearchContent.feature (hunk 5 above)
Comment (bogusez) [reply features/standard/SearchContent.feature]: Done, thx

File: features/standard/SearchContent.feature (hunk 6 above)
Comment (bogusez) [reply features/standard/SearchContent.feature]: Done, thx

**Pull request merged**